eZ/Publish/Core/Search/Legacy/Content/WordIndexer/Gateway/DoctrineDatabase.php

```diff
@@ -136,6 +143,8 @@ public function index(FullTextData $fullTextData)
                     'ContentClassAttributeID' => $fullTextValue->fieldDefinitionId,
                     'identifier' => $fullTextValue->fieldDefinitionIdentifier,
                     'integer_value' => $integerValue,
+                    'language_code' => $fullTextValue->languageCode,
+                    'is_main_and_always_available' => $fullTextValue->isMainAndAlwaysAvailable,
```

Comment:
The `language_code` / `is_main_and_always_available` values needs to added to  https://github.com/ezsystems/ezpublish-kernel/blob/4d462237522b2cbc71843d643cc4e110da528660/eZ/Publish/Core/Search/Legacy/Content/WordIndexer/Gateway/DoctrineDatabase.php#L155-L158 as well.

Reply:
Right, added

Reply:
Right, added